m/mysql-connector-j/mysql-connector-j_ubi_8.5.sh

```diff
@@ -0,0 +1,108 @@
+wget https://search.maven.org/remotecontent?filepath=org/junit/jupiter/junit-jupiter-api/5.8.2/junit-jupiter-api-5.8.2.jar -O junit-jupiter-api-5.8.2.jar
+wget https://search.maven.org/remotecontent?filepath=org/apiguardian/apiguardian-api/1.1.2/apiguardian-api-1.1.2.jar -O apiguardian-api-1.1.2.jar
+wget https://search.maven.org/remotecontent?filepath=org/opentest4j/opentest4j/1.2.0/opentest4j-1.2.0.jar -O opentest4j-1.2.0.jar
+wget https://search.maven.org/remotecontent?filepath=org/javassist/javassist/3.28.0-GA/javassist-3.28.0-GA.jar -O javassist-3.28.0-GA.jar
+wget https://search.maven.org/remotecontent?filepath=com/google/protobuf/protobuf-java/3.19.4/protobuf-java-3.19.4.jar -O protobuf-java-3.19.4.jar
+wget https://search.maven.org/remotecontent?filepath=org/slf4j/slf4j-api/1.7.35/slf4j-api-1.7.35.jar -O slf4j-api-1.7.35.jar
+wget https://search.maven.org/remotecontent?filepath=org/hamcrest/hamcrest/2.2/hamcrest-2.2.jar -O hamcrest-2.2.jar
+wget https://search.maven.org/remotecontent?filepath=com/oracle/oci/sdk/oci-java-sdk-common/2.14.1/oci-java-sdk-common-2.14.1.jar -O oci-java-sdk-common-2.14.1.jar
+
+cd /root
+
+OS_NAME=$(cat /etc/os-release | grep ^PRETTY_NAME | cut -d= -f2)
+
+#Check if package exists
+if [ -d "$PACKAGE_NAME" ] ; then
+  rm -rf $PACKAGE_NAME
+  echo "$PACKAGE_NAME  | $PACKAGE_VERSION | $OS_NAME | GitHub | Removed existing package if any"  
+ 
+fi
+
+if ! git clone $PACKAGE_URL $PACKAGE_NAME; then
+    	echo "------------------$PACKAGE_NAME:clone_fails---------------------------------------"
+		echo "$PACKAGE_URL $PACKAGE_NAME"
+        echo "$PACKAGE_NAME  |  $PACKAGE_URL |  $PACKAGE_VERSION | $OS_NAME | GitHub | Fail |  Clone_Fails"
+    	exit 0
+fi
+
+cd  $PACKAGE_NAME
+git checkout $PACKAGE_VERSION
+
+if ! ant -Dcom.mysql.cj.build.jdk=`which javac | xargs readlink -f | xargs dirname | xargs dirname` -Dcom.mysql.cj.extra.libs=/root/libs dist; then
+    echo "------------------$PACKAGE_NAME:build_fails-------------------------------------"
+	echo "$PACKAGE_URL $PACKAGE_NAME"
+	echo "$PACKAGE_NAME  |  $PACKAGE_URL | $PACKAGE_VERSION | $OS_NAME | GitHub | Fail |  Build_Fails"
+	exit 1
+fi
+
+# Following test failures have been observed which are in parity with intel platform:
+# [junitlauncher] Running testsuite.regression.ConnectionRegressionTest
+# [junitlauncher] Tests run: 165, Failures: 3, Aborted: 26, Skipped: 1, Time elapsed: 344.769 sec
+# [junitlauncher] Running testsuite.regression.SyntaxRegressionTest
+# [junitlauncher] Tests run: 20, Failures: 2, Aborted: 3, Skipped: 0, Time elapsed: 11.819 sec
+# [junitlauncher] Running testsuite.simple.ConnectionTest
+# [junitlauncher] Tests run: 47, Failures: 1, Aborted: 12, Skipped: 0, Time elapsed: 107.953 sec
+
+# Running the test cases require a running MySql server instance. Currently MySql is not supported on ppc64le.
```

Comment:
do the above test cases then pass with this workaround? have we tried checking with mariadb instead of mysql?

Reply:
These test failures are in parity with Intel platform. 
Tried using MariaDB, but similar kind of failures are observed.
Also cannot use MariaDB for the complete test suite as MariaDB does not provide X-Protocol support (port 33060). Hence it will lead to failure of all X-Protocol related tests.